src/actions/guides/database/querying-deleting.cr

```diff
@@ -58,13 +58,11 @@ class Guides::Database::QueryingDeleting < GuideAction
     * `find(id)` - Returns the record with the primary key `id`. Raise `Avram::RecordNotFoundError` if no record is found.
     * `last` - Returns the last record. Raise `Avram::RecordNotFoundError` if no record is found.
     * `last?` - Returns the last record. Returns `nil` if no record is found.
-    * `all` - Returns an array of all the records.
 
     ```crystal
     first_user = UserQuery.first
     last_user = UserQuery.last
     specific_user = UserQuery.find(4)
-    all_users = UserQuery.all
```

Comment:
Could you add this back in, but call `new` instead? Just so people have it there if they need it